src/views/LiteratureOrder.js

```diff
@@ -0,0 +1,310 @@
+import React from 'react'
+import { camelCase } from 'lodash'
+import { formatMoney } from 'accounting'
```

Comment:
Not sure on this package size, you might need to find a way to do the same. If not possible then go to the source and copy the function out of it.